pkg/cmd/grafana/apiserver/server.go

```diff
@@ -101,6 +101,20 @@ func (o *APIServerOptions) Config() (*genericapiserver.RecommendedConfig, error)
 		return nil, fmt.Errorf("failed to apply options to server config: %w", err)
 	}
 
+	if !o.Options.ExtraOptions.DevMode {
```

Comment:
rather than devMode... can this be the existence of:
```
--grafana.authn.signing-keys-url
```
or similar?

Reply:
Sure! That sounds good.